server/controller/messages.js

```diff
@@ -0,0 +1,65 @@
+import models from '../models';
+
+const { Messages } = models;
+
+/**
+ * A class that handles messages methods
+ */
+class messageController {
```

Comment:
Since this is a class let's use `MessageController` instead of `messageController`

Reply:
Okay, would do that.